libs/cli/langchain_cli/integration_template/pyproject.toml

```diff
@@ -7,25 +7,21 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
-langchain-core = ">=0.0.12"
+langchain-core = "^0.1"
 
 [tool.poetry.group.test]
 optional = true
 
 [tool.poetry.group.test.dependencies]
-pytest = "^7.3.0"
-freezegun = "^1.2.2"
-pytest-mock  = "^3.10.0"
-syrupy = "^4.0.2"
```

Comment:
I think we need `syrupy` at a minimum for snapshot flag. Others maybe not?

Reply:
lets remove snapshot flag if not used anywhere